src/ctap2/preflight.rs

```diff
@@ -165,9 +165,8 @@ pub(crate) fn do_credential_list_filtering_ctap2<Dev: FidoDevice>(
                 // This chunk contains a key_handle that is already known to the device.
                 // Filter out all credentials the device returned. Those are valid.
                 let credential_ids = response
-                    .assertions
                     .iter()
-                    .filter_map(|a| a.credentials.clone())
+                    .filter_map(|a| a.assertion.credentials.clone())
```

Comment:
`a` is now a `GetAssertionResult`, right? Might be helpful to make the name more descriptive.